armi/cases/case.py

```diff
@@ -769,10 +770,17 @@ def copyInterfaceInputs(
         # guess. In future, it might be nice to have interfaces specify which
         # explicitly.
         for key, files in interfaceFileNames.items():
-            if isinstance(key, settings.Setting):
-                label = key.name
-            else:
-                label = key
+
```

Comment:
This change seems to have taken away the ability to copy over files which don't correspond to a specific setting. For instance, one may have auxiliary files that support a calculation but are not called out as a specific case setting. 

It seems that we used to be able to copy such files with `copyInterfaceFiles()` in conjunction with a `specifyInputs()` method on the interface in question. But from the changes here, we can no longer do that I guess.

Is this intentional? Because it broke some of our downstream projects and we'll need a way around, if so.

@john-science @jakehader @opotowsky

Reply:
(Also, FYI, this wasn't even my downstream project that was broken, for once! 😆 )

Reply:
@opotowsky Do you have bandwidth to fix this one? 

It seems like there's a use-case that we didn't consider in reviewing this PR.  It happens.

Perhaps we could build tests for both use-cases, so this will be clearer in the future?  (You just Know I love tests.)

Reply:
I can speak to (2).

Often in our models we have some files that are just a big data dump of things like control rod positions at different times, or fuel shuffling operations for each cycle. This data doesn't readily live anywhere in ARMI, and thus needs to be stored in some "auxiliary" file that is parsed and utilized during the run.

I suppose, in theory, both of the examples I listed just now could be stored in either the custom `controlLogic` file or the `shuffleLogic` file, but it typically doesn't make sense to do that based on the size/format of the data.